content/en/quickstart/device-to-device.md

```diff
@@ -11,3 +11,24 @@ menu:
     weight: 20
 toc: true
 ---
+
```

Comment:
I miss a simple Overview where you briefly explain that D2D is possible, it communicates by default over udp. and so on.

Reply:
Still miss this introduction to how does it work. I am pretty sure it's in the core spec.

Reply:
> Still miss this introduction to how does it work. I am pretty sure it's in the core spec.

What do you mean by above sentence? Do you want to add more description at the start of `This guide will walk...` about device core spec ? Could you give more detail direction?

Reply:
I am documenting it already in a different page. ignore.